src/pruna/algorithms/quantization/quanto.py

```diff
@@ -12,7 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Dict
+from typing import TYPE_CHECKING, Any, Dict
+
+if TYPE_CHECKING:
+    from optimum.quanto import Calibration, quantize
```

Comment:
since later we use Calibration and quantize from the imported_modules, we don't need the global imports.
Therefore the TYPE_CHECKING is also not required.

```suggestion
from typing import Any, Dict
```